.github/workflows/conda-build.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will setup a conda build for dependency checks by pulling in the codebuild docker images 
+# and will execute only on modification of yml files for PRs and merges.
+
+name: Build Conda Environment
+
+on:
+  push:
+    branches:
+      - main
+    paths: 
+      - '*.yml'
+  pull_request:
+    branches:
+      - main
+      - feature/**
+    paths: 
+      - '*.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Pull codebuild docker images
+      run: |
+        docker pull public.ecr.aws/codebuild/amazonlinux2-x86_64-standard:3.0
+        docker pull public.ecr.aws/codebuild/local-builds:latest
```

Comment:
Why do we need both of these?

Reply:
The AL2 is the base image for on which we want to build our NBI. The second image is required for running builds locally using the CodeBuild Local Build feature.

Reply:
Do we do both in our github workflow?

Reply:
If we provide only the AL2 build, in the conda-build step the agent requires the local-build image and it pulls it anyways.